src/main/java/net/sf/jabref/cli/SystemOutputPrinter.java

```diff
@@ -0,0 +1,21 @@
+package net.sf.jabref.cli;
+
+import net.sf.jabref.logic.importer.OutputPrinter;
+
+public class SystemOutputPrinter implements OutputPrinter {
+
+    @Override
+    public void setStatus(String s) {
+        System.out.println(s);
+    }
+
+    @Override
+    public void showMessage(Object message, String title, int msgType) {
```

Comment:
The implementation in FetcherPreviewDialog shows the Object as:

``` java
    @Override
    public void showMessage(Object message, String title, int msgType) {
        JOptionPane.showMessageDialog(this, message, title, msgType);
    }
```
so I guess that is the reason for having an Object.